e2e/app/xpath-builders.ts

```diff
@@ -62,7 +62,7 @@ export function buildXPath(xOpts: XPathOptions, container?: Container): string {
     selector = `${textExpr}//${type}`;
     break;
   case ElementType.Dropdown:
-    selector = `${textExpr}${nodeLevel}//*[contains(concat(" ", normalize-space(@class), " "), " p-dropdown ")]`;
+    selector = `${textExpr}${nodeLevel}//*[${classNameContains('p-dropdown')} and not(${classNameContains('p-disabled')})]`;
```

Comment:
How about this approach? Excluding elements that have class name `p-disabled`.
![Screen Shot 2020-09-29 at 10 15 38 AM](https://user-images.githubusercontent.com/35533885/94570523-f1071a80-023c-11eb-9594-72f35cbc298f.png)

Reply:
Thanks for the additional attempt, but I think this is probably slightly worse than the previous approach: p-disabled doesn't relate to the content, and it will likely be removed from these child components in the near future.

One potential fix here is to refactor the help sidebar s.t. we avoid rendering other tabs instead of just hiding them. This would make the implementation better as well. @dolbeew can you think of any issues this might cause?